libs/dh/market-participant/feature-organization/src/lib/overview/dh-market-participant-organization-overview.component.html

```diff
@@ -69,7 +69,7 @@
           </div>
         </ng-container>
         <ng-container *ngIf="element.actor === undefined">
-          <div class="actor-roles-cell-container">
+          <div class="list-cell-container">
```

Comment:
Nitpick: Since `div.list-cell-container` is the only content of the `ng-container`, the if statement could just be applied to the `div` directly and remove `ng-container`.

**Applying this suggestion, remember to remove the `ng-container` to avoid double if statements**

```suggestion
          <div class="list-cell-container" *ngIf="element.actor === undefined">
```